### ISSUE TITLE
Does anybody known proper solution to run this with scaling on raspberry pi?

### ISSUE THREAD

**Issue by HoshiYamazaki** [xerpi/vita-udcd-uvc#24]
As title says.

**Comment by xerpi** [comment]
Have you tries with mplayer or VLC?

**Comment by osvathrobi** [comment]
Not sure about the issue you are having, what I had to do is compile VLC on the raspberry with hardware acceleration enabled - I finished Persona 4 Golden with a smooth frame-rate in fullscreen afterwards.

The default VLC or MPlayer installed from apt does not have proper hw support on raspbian - follow any tutorial out there for manually compiling and have lots of patience while the compiler finishes. (~40 minutes for me)

**Comment by HoshiYamazaki** [comment]
> Not sure about the issue you are having, what I had to do is compile VLC on the raspberry with hardware acceleration enabled - I finished Persona 4 Golden with a smooth frame-rate in fullscreen afterwards.
> 
> The default VLC or MPlayer installed from apt does not have proper hw support on raspbian - follow any tutorial out there for manually compiling and have lots of patience while the compiler finishes. (~40 minutes for me)

Do I need some specific flags for compiling? I tried vitadock-vlc and vitadock-mpv but bot seems to be buggy or miss something.

**Comment by osvathrobi** [comment]
Yes, lots of flags :) Just use a tutorial like this one. There might be libraries missing when you begin to compile so just install them if you get errors. 

https://alignan.github.io/post/compile-vlc-raspberry-stretch/

**Comment by HoshiYamazaki** [comment]
> Yes, lots of flags :) Just use a tutorial like this one. There might be libraries missing when you begin to compile so just install them if you get errors.
> 
> https://alignan.github.io/post/compile-vlc-raspberry-stretch/

I remember when I selected `OpenMAX IL video output` there was a black screen and lag, beside I compiled everything from here:
https://github.com/CrashCortez/vitadock-vlc
I will check when I head home this tutorial, maybe I miss package or so.

**Comment by osvathrobi** [comment]
Just wanted to mention: I used OpenGL output and after compilation and it worked perfectly - no lag, fullscreen video.

**Comment by xerpi** [comment]
Check https://github.com/xerpi/vita-udcd-uvc/issues/40#issuecomment-559259911
